conda-requirements.txt

```diff
@@ -26,11 +25,11 @@ imagehash
 requests
 
 # Optional iris dependencies
-nc_time_axis
-iris_grib
+ecmwf_grib
```

Comment:
I'm not sure whether we should pin this to a specific version of ecmwf_grib as [we did in iris_grib](https://github.com/SciTools/iris-grib/blob/master/environment.yml#L5)?